example/hyperparam/gPyOpt/search_keras.py

```diff
@@ -0,0 +1,146 @@
+import math
```

Comment:
Why is this called search_keras.py instead of just search.py or gpyopt.py? It's not specific to Keras really. I'd just call it gpyopt.py and rename the HyperOpt one in a similar way to make them easier to find.